services/frontend-v3/src/i18n/fr_CA.json

```
@@ -388,18 +386,20 @@
   "profile.hide.account.confirm.content": "Votre compte sera rendu inaccessible, mais les personnes disposant d'un lien direct pourront toujours le consulter.",
   "profile.deactivate.account.confirm.content": "Votre compte sera rendu inaccessible, mais vos données resteront stockées.",
 
-  "profile.addFriend": "Ajouter un ami",
-  "profile.removeFriend": "Supprimer un ami",
-  "profile.removeFriend.confirm": "Voulez-vous vraiment supprimer cet utilisateur de votre groupe d'amis ?",
-
   "profile.is.ongoing": "Courant",
 
   "sign.out": "Déconnexion",
   "home": "Acceuil",
 
+  "profile.addConnection": "Ajouter une connexion",
+  "profile.removeConnection": "Supprimer la connexion",
+
+  "profile.privateGroup": "Groupe privé",
+  "profile.connections": "Connexions",
+
   "profile.visibleCards.public": "Publique",
-  "profile.visibleCards.friends": "Amis seulement",
-  "profile.visibleCards.private": "Privée",
+  "profile.visibleCards.connections": "Connexions",
+  "profile.visibleCards.private": "privée",
```

Not sure why this was changed to lowercase?
```suggestion
  "profile.visibleCards.private": "Privée",
```